openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMBinaryClassificationModelTrainer.java

```diff
@@ -313,7 +313,6 @@ private static void setLightGBMDatasetLabelData(final SWIGTrainData swigTrainDat
             throw new LightGBMException();
         }
 
-        swigTrainData.destroySwigTrainLabelDataArray();
```

Comment:
Yes this is also the same situation as above @mlobofeedzai and should **not** be removed for the same reasons. I'd say, put a comment in the same manner as in the other case so there are no doubts as to why it's there.

I agree with @gandola ,  a try/finally is more idiomatic and better behaved in case of prior exceptions :ok_hand:.